src/vendor/github.com/xelabs/go-mysqlstack/sqlparser/sql.y

```diff
@@ -944,13 +958,15 @@ func forceEOF(yylex interface{}) {
 	lock_algorithm_opts
 	normal_key_opts
 	spatial_key_opts
+    index_options
 
 %type   <indexOption>
 	all_key_opt
 	fulltext_key_opt
 	index_using_opt
 	lock_algorithm_opt
 	normal_key_opt
+    index_option
```

Comment:
Fix format problem. Replace the space with tab.

Reply:
Done. Please check again~